symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/DatafeedService.java

```diff
@@ -0,0 +1,30 @@
+package com.symphony.bdk.core.service.datafeed;
+
+public interface DatafeedService {
+
+    /**
+     * Start the datafeed events service
+     *
+     */
+    void start() throws Throwable;
```

Comment:
I don't really like `throws Throwable`, this is too generic in my opinion.

Reply:
Because inside implementation of `start()` in V1 or even in V2 in future, I use resilience4j-retry to implement the retry mechanism and `Retry#executeCheckedSupplier` throws `Throwable`. So I'm not sure if we have other choices.

Reply:
We could wrap those exceptions
I agree that is feels a bit too generic and it does not tell the user which errors could happen
at least the IllegalStateException if you start it twice

Reply:
So we could wrap the `Retry#executeCheckedSupplier` with a try/catch and checked the type of `Throwable` which got caught, right?